api/operator/v1alpha1/eventing_types.go

```diff
@@ -96,7 +96,7 @@ type EventingStatus struct {
 	BackendConfigHash int64       `json:"specHash"`
 	// Can have one of the following values: Ready, Error, Processing, Warning. Ready state is set
 	// when all the resources are deployed successfully and backend is connected.
-	// It gets Warning state in case backend is not specified and NATS module is not installed or EventMesh secret is missing in the cluster.
+	// It gets Warning state in case backend is not specified, NATS module is not installed or EventMesh secret is missing in the cluster.
```

Comment:
```suggestion
	// It gets Warning state in case backend is not specified, NATS module is not installed, or EventMesh secret is missing in the cluster.
```